utils/build-script-impl

```diff
@@ -810,6 +813,14 @@ if [[ "${CROSS_COMPILE_HOSTS}" ]]; then
     SKIP_COMPILER_RT=1
 fi
 
+if [[ "${BUILD_SWIFT_STDLIB}" ]]; then
+    BUILD_SWIFT_DYNAMIC_STDLIB=1
```

Comment:
Could you add support for `build-swift-sdk-overlay=0`?  Some of our internal presets use it.